tob-web/openshift/angular-on-nginx-deploy.bc-tob.prod.param

```diff
@@ -15,7 +15,7 @@ API_URL=https://api.orgbook.gov.bc.ca/api/v2/
 # BLACK_LIST_CONFIG_FILE_NAME=blacklist.conf
 # BLACK_LIST_CONFIG_MAP_NAME=blacklist-conf
 # BLACK_LIST_CONFIG_MOUNT_PATH=/etc/nginx/
-APPLICATION_DOMAIN=orgbook.gov.bc.ca
+APPLICATION_DOMAIN=angular-on-nginx-devex-von-bc-tob-prod.pathfinder.gov.bc.ca
```

Comment:
I assume this is only required for PROD since we may need to switch between environments when reloading?

Could we give the route a more descriptive name? I know `angular-on-nginx` is the service name, but it is a confusing/ugly URL, what about using something like `frontend` or `tob-web` instead?